### PR TITLE
feat: Change Modern/Default Theme Background to #F8FAFC

### LINKED ISSUE #10533: [Enhancement]: Change the default page background to be closer to white
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Change default page background into a shade of white instead of the current color so that it looks a little more modern. 

Also assess the impact on current widgets.

### Why should this be worked on?

The current default page background looks like older admin panels. Changing it white would help bring a refreshed look to the UI.

### PR DESCRIPTION
**Pull request by dhruvikn**
## Description

Quite simply update modern/default theme background from #F6F6F6 to #F8FAFC

~~Another PR #17346 is raised to update this in system themes. PR #17346 should be merged before this PR to enable testing on DPs.~~ Why this plan changed? Read here - https://github.com/appsmithorg/appsmith/pull/17346#issuecomment-1277976886

Fixes #10533 

## Type of change
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Updated Cypress Tests
- Manually - by building a fresh backend on my local system

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have **~~added~~ updated** tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by jsartisan — DISMISSED** [review]

File: app/client/src/components/ads/ColorPickerComponentV2.test.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const store = mockStore({
         properties: {
           colors: {
             primaryColor: "#553DE9",
-            backgroundColor: "#F6F6F6",
+            backgroundColor: "#F8FAFC",
```
Comment (albinAppsmith) [inline app/client/src/components/ads/ColorPickerComponentV2.test.tsx:28]: There is a PR which is migrating this file to propertyControls folder. Please let me know when the QA is done for this change.
https://github.com/appsmithorg/appsmith/pull/17362

**Comment by dhruvikn** [comment]
Can the QA test this by building a fresh backend on their local system? Unfortunately, testing on DP will not be possible on this PR.

If not, can we merge this directly?

cc: @somangshu @dilippitchika @chandannkumar @kamakshibhat-appsmith @laveena-en

**Review by jsartisan — APPROVED** [review]

**Comment by dilippitchika** [comment]
No, we can ask them to test after this merges to release.

**Pull request merged**